app/assets/javascripts/application.js

```diff
@@ -81,7 +81,9 @@
     specificPage = specificPage.replace(emailPattern, '[email]')
 
     // Preopulate specific page field
-    $('#link').val(specificPage)
+    if (specificPage && !(GOVUK.feedback.getPathFor(specificPage) === '/contact/govuk')) {
```

Comment:
```suggestion
    if (specificPage && !GOVUK.feedback.getPathFor(specificPage).startsWith('/contact')) {
```

I wonder if this is better to capture any potential URL changes?

Reply:
That seems fair, first JS pr so I was going with whatever I could easily find.